tests/python/gaia-ui-tests/gaiatest/apps/system/app.py

```diff
@@ -101,6 +101,16 @@ def wait_for_app_update_to_clear(self):
         Wait(self.marionette).until(lambda m: element.location['y'] == (0 - element.size['height']))
 
     @property
+    def screen_width(self):
+        self.marionette.switch_to_frame()
+        return self.marionette.find_element(*self._screen_locator).rect['width']
+
```

Comment:
Wouldn't it be better if we return to the displayed app frame once we're done with finding the element?  This method can be called at any time, and it'd be better if the user doesn't have to explicitly return to the displayed app frame after calling this method.